extensions/amp-consent/0.1/test/test-consent-config.js

```diff
@@ -542,6 +544,8 @@ describes.realWin('ConsentConfig', {amp: 1}, (env) => {
 
   describe('expandConsentEndpointUrl', () => {
     it('support expansion in allowed list', async () => {
+      getConsentStateManager(doc);
```

Comment:
Why are we doing this here?

Reply:
Incorrectly left over --- assumed necessary stubbing

Reply:
Although it was not really stubbed.